test/test_common/network_utility.cc

```diff
@@ -104,6 +104,11 @@ const std::string addressVersionAsString(const Address::IpVersion version) {
 
 Address::InstanceConstSharedPtr getSomeLoopbackAddress(Address::IpVersion version) {
   if (version == Address::IpVersion::v4) {
+#ifdef __APPLE__
```

Comment:
Everything binds zero, but some test release temporarily after the bind (the bind zero is an allocation without hold). I agree with your suggestion.